docs/.vuepress/config.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
         '/samples/': [
           'basic',
           'labels',
+          'labelsFontsAndColors',
```

Comment:
maybe not just yet, but when 3rd sample is created for labels, those should be placed in a folder

Reply:
yes, always :)